tools/stats/import_test_stats.py

```diff
@@ -83,9 +83,14 @@ def get_slow_tests(
 
 def get_test_times(dirpath: str, filename: str) -> Dict[str, Dict[str, float]]:
     url = "https://raw.githubusercontent.com/pytorch/test-infra/generated-stats/stats/test-times.json"
+    build_environment = os.environ.get("BUILD_ENVIRONMENT")
+    if build_environment is None:
+        test_times=fetch_and_cache(dirpath, filename, url, lambda x: x)
+        raise RuntimeError(f"BUILD_ENVIRONMENT is not defined, available keys are {test_times.keys()}")
 
     def process_response(the_response: Dict[str, Any]) -> Any:
```

Comment:
It's fun to see the param is call `the_response` instead of just the usual `response` :D